parts/k8s/kubernetesagentcustomdata.yml

```diff
@@ -137,7 +137,7 @@ write_files:
     KUBELET_IMAGE={{WrapAsVariable "kubernetesHyperkubeSpec"}}
     KUBELET_REGISTER_SCHEDULABLE=true
     KUBELET_NODE_LABELS={{GetAgentKubernetesLabels . "',variables('labelResourceGroup'),'"}}
-{{if IsNVIDIADevicePluginEnabled}}
+{{if IsNSeriesSKU .}}
```

Comment:
can you keep both of these checks? we want this to be enabled per GPU node and when NVIDIA device plugin enabled, but not enabled with k8s 1.8, 1.9 etc since were are not supporting device plugin in those version.
so it'll look something like:
```
{{if IsNSeriesSKU .}}
{{if IsNVIDIADevicePluginEnabled}}
...
{{end}}
{{end}}
```